src/itest/java/uk/gov/companieshouse/pscdataapi/config/AbstractIntegrationTest.java

```diff
@@ -24,6 +24,6 @@ public abstract class AbstractIntegrationTest extends AbstractMongoConfig {
     public CompanyMetricsApiService companyMetricsApiService;
 
     @MockBean
-    CompanyExemptionsApiService companyExemptionsApiService;
+    public CompanyExemptionsApiService companyExemptionsApiService;
```

Comment:
These fields should be private

Reply:
Will resolve in this PR: https://github.com/companieshouse/psc-data-api/pull/163